.github/workflows/integrate.yml

```diff
@@ -0,0 +1,18 @@
+name: Integrate
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
What's the advantage/reason for running this flow on both PRs and pushes?

Reply:
Probably none.. especially in the current context for sure. Updated.